### PR TITLE
Detecting for Image Uploaded Request

### PR DESCRIPTION
**Pull request by zharmedia386**
+ Generating the image detected into static/detections
+ Building the FrontEnd Side using Flask and JQuery
+ Showing the result in Web App soon after image uploaded
+ Update Readme For Image Uploaded Request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -690,6 +690,16 @@ need image url list json form
...
+### Image Upload (http://localhost:5050)
+The output will be generated and saved into `static/detections/OUTPUT_NAME.png`
+#### Request example
+![](data/helpers/image_upload_request.png)
+
+#### Response example
+![](data/helpers/image_upload_response.png)
+
+<br><br>
+
```
Comment (Qone2) [inline README.md:702]: good! I'll move this part to the top

File: app.py [hunk 2]
```diff
@@ -50,6 +50,11 @@ class Flag:
 print("loaded")
 
 
+@app.route('/')
+def home():
+	return render_template('./index.html')
```
Comment (Qone2) [inline app.py:55]: there is tab indent i'll change it to empty spaces later

File: static/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+window.onload = () => {
```
Comment (Qone2) [inline static/index.js:1]: i'll change tab indent to spaces indent later

**Review by Qone2 — COMMENTED** [review]
Its so cool! Thank you for your contribution! 
I'll change some tab indent to space indent later in my commit.

**Comment by Qone2** [comment]
Its so cool! Thank you for your contribution!

**Comment by Qone2** [comment]
> Its so cool! Thank you for your contribution! I'll change some tab indent to space indent later in my commit.

https://standardjs.com/rules-en.html
https://stackoverflow.com/questions/5430806/what-is-the-indentation-standard-for-html-tab-two-spaces-etc

**Comment by zharmedia386** [comment]
Alright.. Thanks for the changes as well.. Looking forward to collaborate 👍

**Pull request merged**